client/src/components/Workflow/Run/WorkflowRunName.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { BAlert } from "bootstrap-vue";
+import { onMounted, ref } from "vue";
+
+import { getWorkflowInfo } from "@/components/Workflow/workflows.services";
+
+import { errorMessageAsString } from "../../../utils/simple-error";
+
+import WorkflowInformation from "@/components/Workflow/Published/WorkflowInformation.vue";
+
+const props = defineProps({
+    model: {
+        type: Object,
+        required: true,
+    },
+});
+
+const loading = ref(true);
+const messageText = ref<string | null>(null);
+const messageVariant = ref<string | null>(null);
+const expandAnnotations = ref(true);
+const workflowInfoData = ref(null);
+
+onMounted(() => {
+    loadAnnotation();
+});
+
+async function loadAnnotation() {
+    loading.value = true;
+
+    try {
+        const workflowInfoDataPromise = getWorkflowInfo(props.model.runData.id);
+        workflowInfoData.value = await workflowInfoDataPromise;
+    } catch (e) {
+        messageVariant.value = "danger";
+        messageText.value = errorMessageAsString(e, "Failed to fetch Workflow Annotation.");
+    } finally {
+        loading.value = false;
+    }
+}
+</script>
+
+<template>
+    <div class="ui-portlet-section w-100">
+        <div
+            class="portlet-header cursor-pointer"
+            role="button"
+            :tabindex="0"
+            @keyup.enter="expandAnnotations = !expandAnnotations"
+            @click="expandAnnotations = !expandAnnotations">
+            <b class="portlet-operations portlet-title-text">
+                <span v-localize class="font-weight-bold">About This Workflow</span>
+            </b>
+            <span v-b-tooltip.hover.bottom title="Collapse/Expand" variant="link" size="sm" class="float-right">
+                <FontAwesomeIcon :icon="expandAnnotations ? 'chevron-up' : 'chevron-down'" class="fa-fw" />
```

Comment:
Minor, but we now use icons like:
```suggestion
                <FontAwesomeIcon :icon="expandAnnotations ? faChevronUp : faChevronDown" fixed-width />
```

... where the two icons are also imported at the top. (We have examples of this in other components)
I think **in the screenshot** the icons are also flipped where: it should be chevron down for an unexpanded one and chevron up for expanded.